### PR TITLE
TDR 2298 add ordinal to api query

### PR DESCRIPTION
**Pull request by MancunianSam**
Add ordinal to API query
    
This is the ordinal on file property values

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -44,6 +44,11 @@ lazy val akkaHttpVersion = "10.2.10"
 lazy val circeVersion = "0.14.3"
 lazy val testContainersVersion = "0.40.10"
 
+dependencyOverrides ++= Seq(
```
Comment (TomJKing) [inline build.sbt:47]: This is in the `build.sbt` file twice. Also on line 89: https://github.com/nationalarchives/tdr-consignment-api/blob/master/build.sbt#L89. Probably should remove one of them

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment. Thanks

File: build.sbt (hunk 1 above)
Comment (MancunianSam) [reply build.sbt]: I'm blind, I didn't notice it was already in there. I've removed the one I added.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**